src/wordy_webserver_response.js

```diff
@@ -21,6 +23,10 @@ function userOut(storage, userId, callback) {
   });
 }
 
+function ping(storage, userId, callback) {
```

Comment:
Although I understand the reasons of keeping this signature, feels a bit odd to pass three parameters when you only need one.